### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by peeriot** [peeriot/murf#1]
This issue is used to collect general feedback to `murf`. Feel free to write anything :)

**Comment by taladar** [comment]
You should probably release this on crates.io as a <1.0 version. People are unlikely to use something like a unit test framework in their projects (something you build on top of) directly from a git repository.

**Comment by peeriot** [comment]
Good point, thanks. We also thought about this and maybe we should really do this right now, even if the project is in this early stage.
